Bugsnag/Client/BugsnagClient.m

```diff
@@ -165,7 +165,8 @@ - (instancetype)initWithConfiguration:(BugsnagConfiguration *)configuration {
         if (!_configuration.user.id) { // populate with an autogenerated ID if no value set
             [_configuration setUser:[BSG_KSSystemInfo deviceAndAppHash] withEmail:_configuration.user.email andName:_configuration.user.name];
         }
-        
+        [_configuration validate];
+
```

Comment:
This change is causing the unit tests to fail.

By design, `validate` is called from `start` so that an exception won't be thrown too early.